core/src/main/java/com/opentext/ia/sdk/sip/SipSegmentationStrategy.java

```diff
@@ -73,6 +77,40 @@
   }
 
   /**
+   * Return a {@linkplain SipSegmentationStrategy} that allows a maximum total size (uncompressed) of the SIP including
```

Comment:
You should probably mention that the check doesn't take the PDI into account, just so that users are aware of this limitation.